src/Message.js

```diff
@@ -28,11 +28,34 @@ export const MessageTime = ({ time, isBot }: MessageTimeProps) => {
 
 type MessageProps = {
   chat: ChatMessage,
-  onButtonClick?: (title: string, payload: string) => void
+  onButtonClick?: (title: string, payload: string) => void,
+  voiceLang: ?string
 };
-const Message = ({ chat, onButtonClick }: MessageProps) => {
+
+const speak = (message, voiceLang) => {
+  let synth = window.speechSynthesis;
+  let voices = [];
+  voices = synth.getVoices();
+  var toSpeak = new SpeechSynthesisUtterance(message);
+  var selectedVoiceName = voiceLang ? voiceLang : 'en-US';
```

Comment:
I would prefer if this feature is turned off by default. So as long, as a user does not specify the `voiceLang` prop no sound is happening.